### ISSUE TITLE
Consensus in silico target prediction modeling for Series 3 

### ISSUE THREAD

**Issue by spadavec** [OpenSourceMalaria/OSM_To_Do_List#503]
## Overview 

Although Series 4 is the current _de facto_ series of interest, there seem to be some open questions regarding series 3 (S3). For example, it is still unknown which target is hit by series 3 compounds (previous attempts at finding MoAs was focused primarily on arylpyrrole compounds [1] which primarily eliminated PfATP4 as a target, along with a number of predicted targets [Q8I3U9,Q08210, Q8I553, Q8IHS2, P61075]). As such, I've followed my interest here to see if something could be born out in an in silico approach to provide some leads for potential targets. 

Given the number of synthesized S3 compounds, I wanted to develop a consensus model of likely targets for S3; the individual steps for this approach are detailed in this [workbook](http://malaria.ourexperiment.org/in_silico_target_pr/15898/Target_Prediction_Methods.html), but for brevity sake (yet somehow still very long winded!), I will only give an overview here.

## Collection of organism agnostic S3 targets
First, all the S3 compounds were collected and run through a pre-compiled ChEMBL naive-bayes targets prediction model. Predictions were made for each compound at both the 1 and 10microM levels, and targets with at least 80% confidence (of being a hit) were retained. All target hits for the 10microM level can be found [here](https://docs.google.com/spreadsheets/d/10K3r9Zxx8zGzppS1Dwa7cea2vjvlqGCg5LLOrMTCy3o/edit#gid=0), and the 1microM level [here](https://docs.google.com/spreadsheets/d/10K3r9Zxx8zGzppS1Dwa7cea2vjvlqGCg5LLOrMTCy3o/edit#gid=700813132). A sampling of the most represented targets at both levels can be seen below:

CHEMBL_ID	|Organism|	Target Name|	Target Count	|UNIPROT	|Function|	Redundant?|
--	|--|	--|	--	|--	|--|	--|
CHEMBL4462|	Homo sapiens|	NAD-dependent deacetylase sirtuin 2|	23	|Q8IXJ6	|Deacetylase|No|
CHEMBL5554	|Homo sapiens|	Phosphatidylinositol-4-phosphate 3-kinase C2 domain-containing beta polypeptide |	21	|O00750	|Kinase|No|
CHEMBL4226	|Homo sapiens|	Dual specificity protein kinase CLK3	|19|	P49761	|Kinase| No|
CHEMBL4461	|Homo sapiens|	NAD-dependent deacetylase sirtuin 3	| 19  (23) |	Q9NTG7	|Deacetylase	|Yes|
CHEMBL4506	|Homo sapiens|	NAD-dependent deacetylase sirtuin 1|	19|	Q96EB6	|Deacetylase|No|
CHEMBL4224	|Homo sapiens|	Dual specificty protein kinase CLK1|	18 (24)	|P49759|	Kinase	|Yes|
CHEMBL4211|	Helicobacter pylori| 	Carbonic anhydrase 1	|17 (17)	|O24855	|Dehydratase	|Yes|
CHEMBL3886	|Homo sapiens|	Mixed lineage kinase 7|	16|Q9NYL2	|Kinase|No|
CHEMBL4367	|Homo sapiens|	Tyrosine-protein kinase TXK	|16	|P42681	|Kinase|No|
CHEMBL5650	|Homo sapiens|	BR serine/threonine-protein kinase 1	|16	|Q8TDC3	|Kinase|No|
CHEMBL5260	|Homo sapiens	|Serine/threonine-protein kinase TNNI3K |	15	|Q59H18	|Kinase|No|
CHEMBL5973	|Mus musculus|	Carbonic anhydrase 15|	15  (18)|	Q99N23	|Dehydratase	|Yes|
  CHEMBL1944499|	Astrosclera willeyana	| Astrosclerin-3	|15 (15)	|A6YCJ1	|Dehydratase	|Yes|
  CHEMBL3009|	Homo sapiens|	Receptor protein-tyrosine kinase erbB-4|	14	|Q15303	|Receptor Tyrosine Kinase|No|
  CHEMBL5819	|Homo sapiens|	Serine/threonine-protein kinase Nek4	|13	|P51957|	Kinase|No|

**Table 1** -- Most represented targets for both 1- and 10-microM assay levels of naive bayes target search. If a targets was found at both 1- and 10-microM levels, it will be marked 'Yes' under 'Redundant?' column, with second count available in parenthesis in 'Count' column,

Given the adenine-like thienopyrimidine core of S3, its rather unsurprising to see the glut of kinases and NAD-dependent targets in this list (compiled list found [here](https://docs.google.com/spreadsheets/d/10K3r9Zxx8zGzppS1Dwa7cea2vjvlqGCg5LLOrMTCy3o/edit#gid=451705598)). By shear volume, the most represented structure at both levels are three related sirtuin targets which have been implicated as part of plasmodium var gene silencing/regulation [2], potentially though hyrolysis of medium and long chain fatty acyl groups from lysine resiues [3].

## Search for Plasmodium falciparium 3D7 target analogs
Given that all of these targets were found in (largely) _Homo sapiens_, these results need to be translated to _Plasmodium falciparum_ to be useful. Operating under the assumption that our _Homo sapien_ targets share some structural similarity to putative Pf targets, we can search for Pf targets that share high sequence identity and further validate those targets.

The top 7 most represented targets (CHEMBL-4224, -4461, -5973, -1944499, -4211, -4266, and -5554) (admittedly a rather arbitrary cutoff), were then used as templates for plasmodium 3d7 blastp searches. All of the results can be found [here](https://docs.google.com/spreadsheets/d/10K3r9Zxx8zGzppS1Dwa7cea2vjvlqGCg5LLOrMTCy3o/edit#gid=1558571866). A subset of sequences showing the highest similarity/identity (albeit still relatively modest), are shown below:

Query ChEMBL ID | Query Protein Name | 3D7 Match Name | Score | E | Identity | Positives | PDB |
|--|--|--|--|--|--|--|--|
ChEMBL4224 |Dual specificty protein kinase CLK1|   PF3D7_1445400 (CLK1) | 230 | 2.00E-66	| 33%|	53%	|3LLT|
|||CLK3 (PF3D7_1114700)|	125	|1.00E-30|	27%|	46%	||
|||GSK3 (PF3D7_0312400)|	112	|4.00E-27|	27%|	47%||
|||PK7 (PF3D7_1337100) |	105	|2.00E-25|	25%|	45%	||
|||MAPK1 (PF3D7_1431500)|	104	|9.00E-24|	27%|	48%	||
|||SIRA (PF3D7_1328800)|	95.5|	2.00E-22|	29%|	48%	|3U31, 3JWP|
ChEMBL5554 | PI43K C2 domain-containing beta polypeptide | PI3K (PF3D7_0515300) |	131	|2.00E-30	|33%|	50%	|4UWL |
|||PI4K (PF3D7_0509800)|	88.6	|2.00E-17	|28%	|51%|	4WAG|

**Table 2** -- Subset of blastp search results for plasmodium analogs of homo sapien targets. Identities and positive rates are for whole sequence alignment, and PDB column is filled for results that have corresponding PDB structure in Pf.

A number of predictions and alignments were found for CLK3 were found to be similar to the CLK1 results and were left out of Table 2. Overall, the most promising results are CLK1/CLK3 and PI3K/PI4K, with modest blastp and E scores, along with the existence of PDB structures for both.

## CLK1 / CLK3
One of the best overall match across all of the targets was CLK1 (PF3D7_1445400), a CMGC Serine/Threonine kinase. Previous work has been conducted on CLK1,
which indicated that it is primarily localized in the _plasmodium_ nucleus and plays a crucial role
in regulating mRNA splicing and has been identified as a potential kinase
 target for the GSK set of compounds [4], and has been shown to be responsive
 to imidazopyridazine derivatives [5]. Overall, the whole-sequence identity between PfCLK1 and CLK1 is low (33%) and marginal positive match percentages (53%), but shows a general agreement (~0.9A RMSD) in tertiary structure (Figure 1).

![clk1-all-align](https://cloud.githubusercontent.com/assets/3453650/26562889/8b58b276-447f-11e7-8c49-a0da27a42d31.png)
**Figure 1** -- Alignment of PfCLK1 (PDB: 3LLT, Purple) and human CLK1 (PDB: 2VAG, Cyan).

Within the ChEMBL database, the compounds which have IC50 data against human CLK1 (with the S3 core as a substructure) show the well known tricyclic kinase inhibitor motif with a thiophene ring (eg below)

![matchers](https://cloud.githubusercontent.com/assets/3453650/26563311/169de2ca-4485-11e7-854e-ae9a317834db.png)

Currently, the ChEMBL database for hCLK1 has ~286 compounds for which IC50 data is available (available [here](https://docs.google.com/spreadsheets/d/10K3r9Zxx8zGzppS1Dwa7cea2vjvlqGCg5LLOrMTCy3o/edit#gid=1743990306)). Using these compounds, a linear regression on ECFP4 fingerprints with the relevant IC50 data was performed, testing for the ability to approximate IC50 for a handful of S3 compounds (OSM-S-106, OSM-S-132, OSM-S-136, and OSM-S-137). For the test compounds, the potencys are labeled as "IC50" against Pfal, but these should likely be labeled as EC50, but will be referenced as IC50 predictions for consistency sake. 

Overall, the approximations seem to bare out the assumption that hCLK1 is receptive to S3 compounds, and that the _plasmodium_ equivalent could be a potential target, with one notable exception. 

ChEMBL ID | Actual IC50 | Predicted IC50 | Fold Difference | 
|--|--|--|--|
OSM-S-106 | 0.08 | 0.16 | 2 | 
OSM-S-132 | ~40 | 0.2 | 200 | 
OSM-S-136 | 15 | 2.8 | 5.3 | 
OSM-S-137 | 1.7 | 7.2 |  4.2 |
** Table 3 ** Potency prediction for S3 compounds using hCLK1 reference compounds 

While a 2-5x fold difference in potency prediction is acceptable in most cases, a ~200x fold difference is not. Interestingly, the largest margin of error for prediction coincided with the more fragment-like compounds (e.g. OSM-S-129 - OSM-S-134) that have smaller R groups at 1 site: 

ChEMBL ID | Actual IC50 | Predicted IC50 | Fold Difference | 
|--|--|--|--|
OSM-S-129 |  45 | 0.3 | 150 | 
OSM-S-132 |  45 | 0.1 | 450 | 
OSM-S-136 |  45 | 0.3 | 150 | 
OSM-S-137 |  45 | 0.33 | 135  |

**Table 4** Potency prediction for S3 compounds using hCLK1 reference compounds 

Here, all of the actual IC50s were marked as ">40", but were labeled as 45 here for comparison. In the case that the S3 potency data is EC50 data, this large difference could be attributed to any number of mechanisms that prevent these fragments from reaching CLK1 and achieving the predicted potency. Alternatively, this could be an artifact of the approach itself, resulting from the low bit-density in the fingerprints that are being removed from regression due to a high drop-out rate (which I will be investigating next). 

## Next Steps 
pCLK1 will be evaluated further using a rigid docking approach to determine if docking poses and scores can further corroborate it being a potential target. 

The remaining targets will be evaluated the same as CLK1, provided either IC50 data exists, or PDB files can be used to provide further illumination. 

## References
[[1]](http://pubs.acs.org/doi/full/10.1021/acscentsci.6b00086)
[[2]](https://www.ncbi.nlm.nih.gov/pubmed/22379140)
[[3]](http://pubs.acs.org/doi/abs/10.1021/cb200230x)
[[4]](http://www.nature.com/nature/journal/v465/n7296/full/nature09107.html)
[[5]](https://www.ncbi.nlm.nih.gov/pubmed/27721154)

**Comment by MFernflower** [comment]
im away fron my linux machine at the momenet - would adding a methyl inbetwixt the two n atoms of the pyrimidine system help stabilize the binding @spadavec

**Comment by MFernflower** [comment]
im away fron my linux machine at the momenet - would adding a methyl inbetwixt the two n atoms of the pyrimidine system help stabilize the binding @spadavec Can you upload the pdb of s3 hit bound to the kinase?

**Comment by spadavec** [comment]
@MFernflower stabilize in which way? Methylating at the 2 position looks to result in the same as extending the external amine--pushing it further from the binding pocket with essentially no change in binding affinity, unfortunately (methyl added in cyan, OSM-S-106 in blue). 

![methyl](https://cloud.githubusercontent.com/assets/3453650/26763581/f757fa16-4909-11e7-8395-fba30bc74b81.png)

Presuming that we (a) want to keep this core, and that (b) this is the putative binding mode/target, there isn't much space near the core with respect to adding new moieties 

![surface](https://cloud.githubusercontent.com/assets/3453650/26763631/aeb2dee2-490a-11e7-9d70-52dc369dcbc6.png)

Just from a casual look at the pose, it would seem that extending/modifying the sulfyl groups would benefit most, or consider branching off the sulfur directly to access that pocket (apologies for the poor picture, its a bit difficult to find a good angle!).

I've attached the pCLK1.pdb and OSM-s-106 and OSM-S-106-methyl in .mol2 format here 

[pCLK1.zip](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1050535/pCLK1.zip)

**Comment by MFernflower** [comment]
@spadavec i am on vacation so no acess to my normal tools... can you dock to this compound? sorry to be a pain!

NC=1C2=C(N=CN1)C=C(S2)C=2C=C(C=CC2)S(=O)(=O)OC

**Comment by spadavec** [comment]
@MFernflower No worries! This methoxy version looks to dock nearly exactly the same as OSM-S-106 (methoxy in grey, OSM-S-106 in blue) 

![methoxy](https://cloud.githubusercontent.com/assets/3453650/26763889/d2b08370-4910-11e7-857f-710f8c6d4ea5.png)

**Comment by MFernflower** [comment]
looks like it may be worth making!

**Comment by spadavec** [comment]
@MFernflower Docked the two above compounds. They both showed a flipped orientation relative to OSM--S-106 (which is in cyan). Apologies for it being a bit scatterbrained, but it should show the relative binding modes: 

![news](https://cloud.githubusercontent.com/assets/3453650/26765861/b8f359c0-4939-11e7-82c6-f8087e9b42eb.png)

(dimethtyl in grey, furan in purple). Interestingly enough, even if you through the amine on the end (e.g. is exactly the same as OSM-S-106, except for the S/O swap), it *still* will have the (relatively) inverted pose. My guess is that the increased aromaticity of the thiophene is pushing the adenine into the pocket, but I'm not entirely sure why...

**Comment by MFernflower** [comment]
@spadavec indeed very interesting! Looks like the methanesulfonic is the winner! Regarding your last comment about the S/N swap I must quote Jurassic Park:  i'm simply saying that life, uh... finds a way"

**Comment by mattodd** [comment]
Just a note @spadavec - this is very nice work. Will digest the details ASAP, and I encourage others to give their thoughts here, but are you recommending looking at an experimental assay vs CLK1?

I put in a proposal for Series 3 in MMV's call for proposals about MoA a couple of months back, and am waiting to hear (not sure if I shared that, sorry, it was fast). We might get some support for experimental validation, and this analysis could help inform that.

**Comment by mbhebhe** [comment]
Thanks @spadavec for all this insightful info. Like Mat said, we will the digest the details ASAP

**Comment by MFernflower** [comment]
@spadavec do you think it is worth for dr.todd and co to make the sulfonate
since it docks so well?

**Comment by MFernflower** [comment]
@spadavec http://www.nature.com/nature/journal/vaop/ncurrent/full/nature22337.html Very interesting paper! There kinase inhibitors look a bit like our S3 drugs!

**Comment by spadavec** [comment]
@MFernflower in my opinion, its probably not worth it just yet to make the sulfonate. Its likely that S3 hits CLK1 to some degree, but whether thats the actual target is less certain--once we hammer that down, doing more structure based work would make more sense. Also, that PI4K paper looks interesting, and is one of the targets that my method dragged up and I will be looking at after I complete my work on _Sir2a_. Finally, I will definitely try the S/Se swap when my docking experiments for Sir2a finish in a few hours. 

@mattodd Regarding a CLK1 assay, I guess I'd say...perhaps? Overall, my intention was to put together 3-5 targets that might merit worth looking at more in depth. Given the number of kinase hits that came up, it might not be worth it to test any one specifically (although if we were, CLK1 would be my first bet) given that the S3 compounds are very likely to be rather promiscuous and hit a few kinases. I'm currently putting together a suite of plasmodium kinase PDB targets that we can dock all of the S3 compounds against and see which targets are most receptive, which would hopefully give us a better idea of which kinase(s) to look at.

**Comment by MFernflower** [comment]
As always fantastic work vito!

**Comment by spadavec** [comment]
@MFernflower Yes; it looks like I should be able to get both PI4K and PI3K; I should have a relatively complete list of kinases by tomorrow evening.

**Comment by mattodd** [comment]
Great work @spadavec. (For onlookers/OSM first-timers, one of the many very good things about what @spadavec is doing is to include justification for all conclusions by way of links to spreadsheets/lab books. No emails were created or harmed in the course of this work.)

A quick mention about the Series 3 background. As a group we decided to move onto Series 4 as a strategic move - S4 had more associated data and proven _in vivo_ activity. By contrast S3 had proven challenging to functionalise without loss of potency. _But S3 remains very interesting_. The quality of the original hit, OSM-S-106, is striking. In response to MMV’s call for proposals around discovering mechanism of action I submitted S3 (file)

[MoA Proposal OSM Series 3.xlsx](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1059567/MoA.Proposal.OSM.Series.3.xlsx)

...since this required only a summary of key data about the relevant compound - my bad for not sharing that file before. I’m hoping MMV agree to look at the MoA for us because OSM-S-106 is a beauty. @mbhebhe has started a PhD recently and can look at S3 and pursue the most interesting lines of inquiry that we as a community determine. It’s always more productive to pursue synthetic targets with strong rationale, and @spadavec 's analysis provides that. We had been thinking about inverting the thieno ring, for example (so, S facing south instead of north) and this has popped up in the analysis. We had also assumed we needed to make the furan and pyrrole analogs of the thiophene, but the analysis cautions against that (with obvious caveats). We might target one of these to see if the prediction holds. With some more work we'd be in a good position to publish everything - not quite there yet since all potencies fell off the cliff - and pursue the series more generally.

I understand you’re still going on this @spadavec (particularly the pSir2a docking, and [PI4K](https://www.nature.com/nature/journal/v504/n7479/full/nature12782.html)/PI3K are interesting targets) but I wanted to comment on formulating some specific to do items now, given that @mbhebhe is well-placed to act, experimentally, on your predictions. We have plenty of 106 to ship to whomever. (@mbhebhe - if any of the below things are large and complex to deal with, you can start new Issues, referencing this one) So (and anyone else feel free to chip in, here, obi).

1) The CLK1 crystal structures in the PDB - who did these and could we work with them? @mbhebhe

2) Similar compounds in the GSK set (ref 4 above) - I’d thought we had extracted all the most relevant, similar compounds. Are there some more in there that we might consider as being S3? I guess we never documented this search properly on the [wiki](http://www.openwetware.org/wiki/OpenSourceMalaria:GSK_Amino-thienopyrimidine_Series#Existing_Analogs).

3) Table 3 is most interesting, and the outlier could indeed be there for reasons not associated with drug-protein interactions. My first thought here, too, is that we still have relatively few actives in S3, meaning 3 out of 4 is pretty good.

4) The docking studies obviously successfully predict the sensitivity of the potency of the S3 compounds to variation in substitution pattern of the primary amine and the nature of the Ar ring. That's most promising.

5) The structures proposed as possible synthesis targets: @mbhebhe is looking into these for feasibility/precedence. Some very nice suggestions there, both simple (open-chain functionalised thiophene) and challenging (requiring direct thiophene-heterocycle couplings). Let’s see which are the lowest-hanging fruit, unless there is a reason to prefer one over another?

6) @mbhebhe - who is running, or has recently run, a CLK1 assay (human or Pfal)? It might be possible to engage Novartis on the PI4K/PI3K possibilities if they remain interesting.

7) Should we bite the bullet and run a [commercial kinase scan](https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling)? Although I guess the kinase profiling in the [2017 paper](http://www.sciencedirect.com/science/article/pii/S0223523416307978) you found was probably done by [ManRos therapeutics](http://www.manros-therapeutics.com/).

**Comment by MFernflower** [comment]
If the kinase scan can do both human and plasmodium kinases and is a good price than I would recommend doing it. If discoverx is too pricy there are always others options @mattodd

**Comment by mbhebhe** [comment]
Once again, thanks @spadavec for doing all this work. You attached links to some google docs, could I please have permission to view them, thanks!

A while back, I came across a few patents that made molecules with the S3 core but different substituents/groups attached to the core. When I looked at them then, they were of no interest but now they might as they look similar to @spadavec 's predicted molecules. I will compile a list and upload soon (bit busy in the lab at the moment). But these similar molecules were useful as TIE-2 and/ or VEGFR-2 inhibitors, useful for modulating the activity of lipid kinases including PI3K, and for treating disorders such as cancer mediated by lipid kinases. An some were EP2 receptor modulators.

![image](https://user-images.githubusercontent.com/26862286/26931565-4084edfa-4ca4-11e7-998f-9668f4f7c2dc.png)
The above molecule is possible to make. Will look into the others. Some looked challenging to synthesise, but will do more searching

People who can do PfCLK1 assays:
- Laurent Meijer (<meijer@manros-therapeutics.com>), paper: Exploration of the imidazo[1,2-b]pyridazine scaffold as a protein kinase inhibitor (http://dx.doi.org/10.1016/j.ejmech.2016.09.064)
- Maybe these guys, paper: Two Nucleus-Localized CDK-Like Kinases With Crucial Roles
for Malaria Parasite Erythrocytic Replication Are Involved in Phosphorylation of Splicing Factor (DOI 10.1002/jcb.23034)

**Comment by spadavec** [comment]
@mattodd Sorry for the long response time--got tied up with other work! 

> The CLK1 crystal structures in the PDB - who did these and could we work with them? @mbhebhe

This structure is [3LLT](http://www.rcsb.org/pdb/explore.do?structureId=3LLT), via the Structural Genomics Consortium (http://www.thesgc.org/). This particular PDB is quite old, but it looks like the people on the PDB submission are still looking at malaria (they have published PfBromodomain structures this year). 

>  Table 3 is most interesting, and the outlier could indeed be there for reasons not associated with drug-protein interactions. My first thought here, too, is that we still have relatively few actives in S3, meaning 3 out of 4 is pretty good.

Agreed. The best data for that comparison would be IC50 data, not EC50 data. However, even with all the extra problems and worries with EC50 data, this is still a very good (albeit very cursory/broad) look. 

> The docking studies obviously successfully predict the sensitivity of the potency of the S3 compounds to variation in substitution pattern of the primary amine and the nature of the Ar ring. That's most promising.

Also agreed. After looking at PI3K/PI4K, I'm going to make a quick diagram that shows which substitutions are acceptable and which aren't, and _might_ be able to translate that to a pharmacophore model, assuming that the goal is to replicate the ATP binding mode.

**Comment by MFernflower** [comment]
looks like our magic bullet is related to a commercial CLK inhibitor! http://www.sigmaaldrich.com/catalog/product/sigma/t5575?lang=en&region=US


can we send some of this off for antimalarial potency testing? @mattodd @spadavec

**Comment by drc007** [comment]
Is that the most stable tautomer? I would've expected
![temp](https://user-images.githubusercontent.com/7080516/27105504-b8f36a28-5087-11e7-9730-dc65ab3a1a6e.png)

**Comment by MFernflower** [comment]
oh dear! thats what a late night does to you! you are 100% right @drc007 !

**Comment by mbhebhe** [comment]
@MFernflower @drc007 
![image](https://user-images.githubusercontent.com/26862286/27113767-57d91a1e-5102-11e7-9e3d-a930c5921e77.png)
The above molecule is possible to make, I did some searching and it can be made from these two methods:
![image](https://user-images.githubusercontent.com/26862286/27113810-a448d736-5102-11e7-8457-8f1ce976cd3f.png) 
from http://onlinelibrary.wiley.com/store/10.1002/anie.201403082/asset/supinfo/anie_201403082_sm_miscellaneous_information.pdf?v=1&s=373324c86f801e598b6d9c18bc63627320ee5f4c

![image](https://user-images.githubusercontent.com/26862286/27113824-b6af7df8-5102-11e7-9462-b4a5a519c0f9.png)
from http://www.journal.csj.jp/doi/pdf/10.1246/bcsj.59.3097

We have to use the amino-thienopyrimidine without the Br as the starting material

**Comment by mattodd** [comment]
Great contributions, everyone.

1) @MFernflower is that commercial compound real? i.e. can it be bought? can we get a quote? Is it available from anywhere else? It's an interesting catch, but significantly different from our S3 compounds, obviously, and would seem to lack the binding motifs that Vito predicted were necessary.

2) @mbhebhe - we'd need to be able to lithiate the full thienopyrimidine, but I guess you know we can do that with the current synthesis that involves a bromination. Do we have the epoxide?

3) There were some excellent Twitter inputs from Stuart Ralph who says i) [There are Pfal sir2 knockouts available](https://twitter.com/sturalph/status/872847369624145921), and [he or Chris Tonkin could test compounds](https://twitter.com/sturalph/status/872847663510675456), and ii) [Christian Doerig and Gabriel Pradel have assays for Pfal CLKs](https://twitter.com/sturalph/status/872848325879316483) - shall I reach out?

4) @spadavec your newer results suggest enough rationale for evaluation against PI3K and PI4K, do you think? I've not checked how we might do that yet.

**Comment by MFernflower** [comment]
@mattodd Sigma has the commercial kinase inhibitor but it's pricey: http://www.sigmaaldrich.com/catalog/product/sigma/t5575?lang=en&region=US

**Comment by mbhebhe** [comment]
@mattodd sadly I have to lithiate again. We have the epoxide in the fridge somewhere and so do the stores.

**Comment by MFernflower** [comment]
@mbhebhe I'm a tiny bit lost - could you draw out the synthetic route to the compound Me and Chris Swain suggested?

**Comment by spadavec** [comment]
@mattodd 

> There were some excellent Twitter inputs from Stuart Ralph who says i) There are Pfal sir2 knockouts available, and he or Chris Tonkin could test compounds, and ii) Christian Doerig and Gabriel Pradel have assays for Pfal CLKs - shall I reach out?

Based on the data that we do have, I would suggest that if we do go this route, the Pfal CLKs (specifically CLK1 and CLK3) tests should be prioritized.  

> your newer results suggest enough rationale for evaluation against PI3K and PI4K, do you think? I've not checked how we might do that yet.

I'm currently looking into relevant assays here for Pfal PI3/4K, but my guess is that the S3 compounds which hit CLK (or any other kinase) are going to show (at least) nominal activity against PI4K (and I'm pretty confident that a fair number will), and that the real confirmation will likely be in the form of a knockdown-like experiment.

**Comment by mbhebhe** [comment]
@MFernflower 
![image](https://user-images.githubusercontent.com/26862286/27173143-f9006066-51fa-11e7-8a3f-703caeaacab5.png)

**Comment by MFernflower** [comment]
Two questions about that pathway:

Doesn't the free amine need to be boc-ed?
Why not go for PCC in CH3CN as the oxidant? 

@mbhebhe

**Comment by holeung** [comment]
I just saw this thread. Great work, a lot of nice ideas to bounce around. Our lab does quite a bit of computational and crystallographic work with human protein kinases, but I'm not familiar with the malarial systems. I suggest looking through the literature to see what Pf kinases are considered promising drug targets. Most kinase inhibitors are promiscuous from a small to shocking degree. The active S3 compounds may hit Pf CLKs, but that might not be related to their mechanisms of action. As @spadavec suggested, it would be useful to do computing screens comparing predicted SARs against CLKs vs other Pf kinases. Also, be careful when using chemical probes like TG003 unless their specificity is thoroughly documented. It's probably hitting all kinds of other kinases along with who-knows-what.

@spadavec: Can you please describe how your target prediction algorithm works?

**Comment by spadavec** [comment]
@holeung The actual ligand/target prediction framework comes from a set of pre-constructed naive bayes classification models from ChEMBL (you can read about it [here](http://chembl.blogspot.com/2016/03/target-prediction-models-update.html); essentially the models are based off all of the existing ligand/protein pairs in the chembl database with some caveats, and a given "test" ligand can be assigned probabilities of being a member of a protein/ligand class (e.g. being a binder) given fingerprint similarity. I did that for most of the S3 ligands, found the most represented targets, and translated those human hits to their plasmodium equivalents.

**Comment by holeung** [comment]
OSM-S-123 is OSM-S-106, the high activity molecule, but without the sulfonamide.

![106and123](https://user-images.githubusercontent.com/10951570/27255633-a109bad0-533d-11e7-9528-865075734904.png)


OSM-S-123 has been annotated as an inhibitor of human IKK beta (nuclear factor kappa B kinase beta subunit), EGFR (epidermal growth factor receptor), and VEGFR1 (vascular endothelial growth factor receptor) under ChEMBL entry [311108](https://www.ebi.ac.uk/chembldb/index.php/compound/inspect/CHEMBL311108).  Of these three kinases, IKK beta has the highest homology to a Pf kinase, CAMK/CDPK. These are heavily studied plasmodium kinases, essential for its life cycle, and considered good drug targets.
https://www.nature.com/articles/ncomms1558
http://www.future-science.com/doi/abs/10.4155/fmc.12.183

[Blast_IKKB_HUMAN.pdf](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1082726/Blast_IKKB_HUMAN.pdf)

Two high homology Pf kinases, CDPK2 and CDPK4, have crystal structure available. http://www.rcsb.org/pdb/explore.do?structureId=4mvf
http://www.rcsb.org/pdb/explore.do?structureId=4qox

I docked OSM-S-106 into both 4mvf and 4qox. This was done with smina with the vinardo scoring function and flexible side chains. Only 4qox gave a reasonable score, of 10.3. I would have expected a higher score given the potency of OSM-S-106. I haven't had the chance to look closely at SAR of Series 3 with these docked structures. The binding pockets are small and probably can't accommodate some of the larger Series 3 ligands that were found as active.

[smina_dock1_4qox.zip](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1082731/smina_dock1_4qox.zip)

In conclusion, I am not convinced that the CDPKs are the true targets either. At this point, it is likely that OSM-S-106 acts as an ATP-mimic, perhaps at a kinase. The Pf kinases are promising targets. OSM may also want to consider a target-oriented strategy in the future. The CDPKs crystal structures came from the Structural Genomics Consortium in Toronto/Oxford. I imagine they would collaborate with OSM.

**Comment by madgpap** [comment]
@spadavec excellent work; I'm glad my models are put to good use. FYI, we followed a very similar approach for target convolution of mTB phenotypic hits - see here: 
http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003253
http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0121492

**Comment by MFernflower** [comment]
@spadavec @mattodd Here are some modifications I thought up of the lead compound. Some may have already been shown while some are totally new:
![kinasemods](https://user-images.githubusercontent.com/3164942/27258008-ba0685da-53bb-11e7-8995-f6aff7fe7195.png)

**Comment by mbhebhe** [comment]
WOW! Thanks for all the great work @spadavec and welcome @holeung 
So am I safe to say CLK1 and CLK3 are promising, Sir2a is a no go, Plk4 is a maybe, CDPKs are a maybe no?

@MFernflower yeah, I would have to protect the amine, thanks. PCC is also another oxidant we could use, I will see what we have available

**Comment by holeung** [comment]
@MFernflower: If the target is intracellular, like most kinases, you want your compound to be charge-neutral at physiological pH. Otherwise, it will be less membrane permeable.

**Comment by MFernflower** [comment]
that is true but my reasoning behind some of them was to mimic atp as close as I can

**Comment by mattodd** [comment]
Excellent new inputs, guys. A quick note: I'm going to loop in the SGC guys ([Ray Hui](http://www.thesgc.org/profile/toronto/rhui), specifically) to scan the targets mentioned here and see if any could be looked at experimentally.

**Comment by holeung** [comment]
If we can experimentally confirm that the MOA targets are protein kinases, I can co-write a grant application on Series 3.

**Comment by holeung** [comment]
I redid the docking of OSM-S-106 vs Pf crystal structures, using smina, Vinardo score, and flexible side chains.

PDB  	Score
4qox	-10.3
4mvf	-8.7
3ltt	       -11.8
5kbf	       -13.0
2yog	-11.4
2bsx	       -9.4
1v0p	-11.4
5lm3	cannot dock
5tpx  	-9.7
5llt   	cannot dock

Top scores go to 5kbf (PKA-R), 3ltt (CLK1), 2yog (thymidylate kinase), 1v0p (PfPK5). I will put models corresponding to these docked complexes shortly.

Our university has a small pilot grant for infectious disease research, with letter of intent due Jan. If we can narrow down our hypotheses, and come up with simple, testable experiments, I'll write the application. Collaborators welcome!

**Comment by MFernflower** [comment]
thymidylate kinase is a top target eh? I wonder if
acyclovir kills plasmodium?

>

**Comment by holeung** [comment]
Docked complexes attached as zipped pdb files.

[5kbf_docked.zip](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1363623/5kbf_docked.zip)
[3ltt_docked.zip](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1363624/3ltt_docked.zip)
[2yog_docked.zip](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1363625/2yog_docked.zip)
[1v0p_docked.zip](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1363626/1v0p_docked.zip)

**Comment by anyachan** [comment]
We have used our alignment-based target prediction model to identify the potential targets of OSM-S-106. Also we identified several kinases (PI3-kinase, CDK2 and CDK5) and carbonic anhydrase as potential targets. A further interesting candidate target is the M18 aspartyl aminopeptidase. 

![image](https://user-images.githubusercontent.com/11487245/33671006-c0e46ed6-daa6-11e7-86bc-05991cc2acdd.png)

**Comment by drc007** [comment]
@anyachan This looks interesting, have you published details of the method?

**Comment by anyachan** [comment]
@drc007 No, we have not published the method. We are planning to do this some time later.

**Comment by drc007** [comment]
@anyachan Hi, Would it be worth considering overlay of maximum common substructure as the first step rather than centre of mass?

**Comment by anyachan** [comment]
@drc007 Hi, yes, it would be definitely worthwhile looking into the possibility of adding a MCS search as an initial step to the algorithm.

**Comment by mbhebhe** [comment]
Hi @anyachan thanks for this. Sorry about the very delayed response, I was on holiday and now I'm back. 
Same as @drc007 's response, I was wondering if there was raw data relating to this, published somewhere . And how can we access it? ELN?

**Comment by mattodd** [comment]
Quick housekeeping question @spadavec . I was just looking at all the above (because we're gathering momentum on writing up Series 3 MoA) and there's a link to a [workbook](http://malaria.ourexperiment.org/in_silico_target_pr/15898/Target_Prediction_Methods.html) up top but the workbook appears to be missing. Can you let me know if it's hidden, or removed? Trying to capture all the working/methods/inputs.

**Comment by spadavec** [comment]
@mattodd I'm currently stuck traveling, but should be able to look into this by Saturday at the latest.
